packages/integration-tests/test/mergeUtxoTest.js

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
need to add this file to these
`integration-tests/ci-baseline-test.js`
`integration-tests/sequence-test.js`